go.mod

```diff
@@ -1,34 +1,34 @@
 module github.com/openshift/cluster-monitoring-operator
 
-go 1.14
+go 1.15
```

Comment:
Why do we need this, does something require 1.15 go version?

Reply:
AFAIK, we don't particularly need it, I just bumped to 1.15  to keep the version up to date. Also, upstream projects are already using 1.15, so I don't think CMO should lack behind.
Do you have any concerns with this upgrade? I can remember past discussions about this topic but after double checking, 1.15 should be available starting from 4.6.

Reply:
This does not bump the version, just states the minimum required version this project needs to compile its deps. If there is no need to require the bump to 1.15, I would say lets use 1.14 instead.